### ISSUE TITLE
Like / Reply placement on comments

### ISSUE THREAD

**Issue by lovebug0706** [lightbeam24/CustomTube#102]
The placement is the same for all the versions, even though the placement changed.
Heres how it looks for all themes. (note the reply, like placement, and the number color) 
![Screenshot 2023-10-17 110954](https://github.com/lightbeam24/CustomTube/assets/34926216/b0159f8d-f555-45d6-8806-24cd011acfb2)
Heres how it looked back in 2009 / 2010. The likes and reply button were on the very right of the comment.
![Screenshot 2023-10-17 110911](https://github.com/lightbeam24/CustomTube/assets/34926216/5fd677cf-7ae2-421f-8753-a7612ca32e8d)
In 2012, it looked like this, and the like counter color was green. I dont know if it was like this in 2009 because i couldnt find any comments with likes.
![Screenshot 2023-10-17 111604](https://github.com/lightbeam24/CustomTube/assets/34926216/41f8d9b2-bc36-48da-9603-0ad126c3b9b6)
In 2013, the reply and like buttons finally moved down to the bottom, but the counter was still green.
![Screenshot 2023-10-17 111710](https://github.com/lightbeam24/CustomTube/assets/34926216/c1474a54-9516-49c3-bc50-3492163762d1)
Im gonna take a guess because comments fail to load on Archive after around 2014 and say that in 2017, the counter finally became blue.

**Comment by lightbeam24** [comment]
I'm well aware of what the old comments looked like, but thanks for the screenshots anyway. Once again, this is something that just hasn't been done yet. (A good rule of thumb is that every CT inaccuracy is something I'm aware of, but haven't gotten to yet) In terms of layout improvements, it's relatively high priority, but code cleanup is top priority right now (which will come with layout improvements elsewhere), so it could be a little while yet before I get to this.

> Im gonna take a guess because comments fail to load on Archive after around 2014 and say that in 2017, the counter finally became blue.

I don't think so. I'm almost 100% certain that they've been blue since at least 2016. Maybe even 2015. Maybe even 2014.

**Comment by lovebug0706** [comment]
Probably 2015, that 2013 one was from 2014, on a snapshot that did work with comments. I was using DJ Earworm's United State Of Pop mashups.

**Comment by lightbeam24** [comment]
Any way you could send a link to the specific archive? I've never found an archive after mid 2013 that works with comments (until Polymer of course). Also DJ earworm seems to have several of these videos, all of which seem to be pretty popular and have lots of archives, so finding that specific archive is going to be like finding a needle in a haystack.

Here are a couple examples of archives of that channel's videos that don't load comments:

https://web.archive.org/web/20141208010626/https://www.youtube.com/watch?v=BjYWwZYLYEs
https://web.archive.org/web/20140508210209/http://www.youtube.com/watch?v=iNzrwh2Z2hQ

**Comment by lovebug0706** [comment]
2010: https://web.archive.org/web/20100113175330/https://www.youtube.com/watch?v=iNzrwh2Z2hQ
2012: https://web.archive.org/web/20120201150135/http://www.youtube.com/watch?v=iNzrwh2Z2hQ
2013: https://web.archive.org/web/20130203031241/https://www.youtube.com/watch?v=iNzrwh2Z2hQ
Your theme: https://www.youtube.com/watch?v=wfTv2WZZMpU
Here you go, had to look through my history.

**Comment by lightbeam24** [comment]
Oh no, I was talking about the 2014 archive that had comments. Unless you were mistaken, and there was no 2014 archive with working comments (which wouldn't surprise me).

**Comment by lovebug0706** [comment]
Yeah i was mistaken, i was thinking of the 2013 one, but it has actually been blue since late 2013! 
![la-fi-tn-youtubes-new-comments-5-things-to-kno-001](https://github.com/lightbeam24/CustomTube/assets/34926216/e193f6cc-4ef2-4e40-887a-9e9d80e4a3ca)
 This image is from an article in November 2013 talking about the petition to remove the Google+ requirement to comment. Link to article: https://www.latimes.com/business/technology/la-fi-tn-youtube-comments-46000-petition-20131108-story.html
